src/transactions/TransactionFrameBase.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2014 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#pragma once
+
+#include "overlay/StellarXDR.h"
+#include <unordered_set>
```

Comment:
NB: I had to add
`#include "ledger/LedgerHashUtils.h"` here as to avoid link errors.
I suspect there are some weird template instantiation issues with `insertKeysForFeeProcessing`